test/EndToEnd/NuGet.Tests.psm1

```diff
@@ -31,6 +31,10 @@ if (!(Test-Path $nugetExePath))
     $nugetExePath = "$nugetRoot\test\EndToEnd\NuGet.exe"
 }
 
+# Enable NuGet Test Mode
```

Comment:
this covers the functional tests, but not the commandline unit tests.

I think we should fix them too, the rest of the tests should be refactored not to use nuget.org at all